app-server/src/main/resources/application.conf

```diff
@@ -0,0 +1,17 @@
+akka {
+  loglevel = DEBUG
+}
+
+http {
+  interface = "0.0.0.0"
+  port = 9000
+}
+
+database = {
+  url = "jdbc:postgresql://database.raster-foundry.internal/rasterfoundry"
+  url = ${?POSTGRES_URL}
+  user = "rasterfoundry"
+  user = ${?POSTGRES_USER}
+  password = "rasterfoundry"
+  password = ${?POSTGRES_PASSWORD}
```

Comment:
Can the hardcoded url/user/passwords be removed? Looks like they'll be overwritten anyway.